### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by r-jeng**
@chiakat @kyang9021 @afong888 

Could someone help code review whenever they have free time? The current product cards should match the  4.1 Related Product Cards 1/3 Trello card staged for review. Thanks!

### CODE REVIEW

**Comment by kyang9021** [comment]
@r-jeng Could you handle the merge conflicts with the current main

**Comment by r-jeng** [comment]
@kyang9021 
Merge conflicts have been resolved!

File: client/src/components/Related/ProductCarousel.jsx [hunk 1]
```diff
@@ -66,27 +77,60 @@ class ProductCarousel extends React.Component {
...
+
+        <button className="carousel__button carousel__button--left" type="button" onClick={() => this.prev()}>
+          <img src="./images/arrow-left.png" alt="" />
+        </button>
+
+        <div className="carousel__track-container">
+          <ul className="carousel__track" ref={this.myRef}>
+            {productInfo.map((product, i) => (
+              <li className="carousel__slide" key={i}>
+                <div className="card">
```
Comment (kyang9021) [inline client/src/components/Related/ProductCarousel.jsx:112]: Consider breaking down each carousel item into an individual file in the future

**Review by kyang9021 — COMMENTED** [review]
Nice job with the carousel!

File: client/src/components/Related/ProductCarousel.jsx (hunk 1 above)
Comment (r-jeng) [reply client/src/components/Related/ProductCarousel.jsx:112]: Noted! Thank you for reviewing!

**Pull request merged**